unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/WorldChatWindowHUD/WorldChatWindowController.cs

```diff
@@ -168,6 +174,7 @@ public void Dispose()
         chatController.OnJoinChannelError -= HandleJoinChannelError;
         chatController.OnChannelLeaveError += HandleLeaveChannelError;
         chatController.OnChannelLeft -= HandleChannelLeft;
+        chatController.OnAskForJoinChannel -= HandleAskForJoinChannel;
```

Comment:
do we need to add also `rendererState.OnChange` unsubscriction if we are still waiting for it?

Reply:
+1